
string from
string name
string to 

---
string path_to_log
